### ISSUE TITLE
Question about model training.

### ISSUE THREAD

**Issue by jarucezh** [hytseng0509/CrossDomainFewShot#33]
Hello Dr. Tseng, Thanks for releasing your code for LFT. I am introducing your repo to reimplement the paper. I wonder whether you use some tricks, for example learning rate decay, to train the model or not. And which model do you utilize to test, the "best_model.tar", "400.tar" or something else? Thanks for your help.

**Comment by ideal-123** [comment]
Hello,  jarucezh, Have you found how to choose model in test time  yet? I have the same question about which model used in test time, cause I found in my experiment that not a definite training model can always perform best in each test dataset. Especially I don't know how to document the results in Table 1.

**Comment by jarucezh** [comment]
Hi @ideal-123, I didn't find the way to choose the model. Sorry...
